src/uhi/numpy_plottable.py

```diff
@@ -206,5 +323,8 @@ def ensure_plottable_histogram(hist: Any) -> PlottableHistogram:
             # Standard tuple
             return NumPyPlottableHistogram(*(np.asarray(h) for h in hist))
 
+    elif hasattr(hist, "InheritsFrom") and hist.InheritsFrom("TH1"):
```

Comment:
TProfile does not insert from TH1, correct?

Reply:
I didn't consider that, but it does - I added support for it (with the different error types etc. using the arrays directly from numpy isn't as simple in this case, unfortunately, so I just used `GetBinContent` and `GetBinError`)